tests/Integration/ORM/EntityProxyFactoryRelationshipTest.php

```diff
@@ -50,6 +51,20 @@ public function doctrine_proxies_are_converted_to_foundry_proxies(): void
         $this->assertInstanceOf(StandardCategory::class, $category);
     }
 
+    /**
+     * @test
+     */
+    public function it_can_set_value_of_a_many_to_one(): void
```

Comment:
As suggested in slack, could you add a test for one-to-many (adding a proxied object to a collection)?